rfc/0009-audit-checks.md

```diff
@@ -0,0 +1,171 @@
+
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Audit checks                           |
+| Start Date   | 08/05/2022                        |
+| Category     | enhancement,feature                 |
+| RFC PR       | https://github.com/kubewarden/rfc/pull/10  |
+| State        | **IN REVIEW**                     |
```

Comment:
```suggestion
| State        | **ACCEPTED**                     |
```

Also, the filename is `0009-audit-checks.md`, and number 9 is already taken by the helm chart RFC (sorry).

Reply:
The file has been renamed